index.html

```diff
@@ -222,7 +222,28 @@ <h2 class="title">About<br>Myself</h2>
                         <hr>                        
                     </div>                   
                 </div> 
-            </section>      
+            </section>   
+            <!--Contac form starts here-->
+            <section class="contac_me">
+                <div class="text_area_contac">
+                    <h3 id="contact_title" >Contact me</h3>
+                    <p id="contact_text">
+                        If you have an application you
+                        are interested in developing, 
+                        a feature that you need built or a
+                        project that needs coding. I'd
+                        love to help with it 
+                    </p>
+                </div>                
+                <form action="https://formspree.io/f/xqknnnpl" method="post" class="form_area_contact">
+                   
+                    <input type="text" name="name" id="name_contact" maxlength="30" placeholder="Full Name" required>
+                    <input type="text" id="email_contact" placeholder="Email" required>
```

Comment:
According to the project requirements, this `<input>` should be of `type="email"`.

<img width="231" alt="Screen Shot 2022-05-31 at 16 28 40" src="https://user-images.githubusercontent.com/82426682/171287545-9fc72aad-a246-4807-8b55-f46edea5d57a.png">

Kindly fix this.